server/start.go

```diff
@@ -340,6 +344,11 @@ func startInProcess(ctx *server.Context, clientCtx client.Context, appCreator ty
 		app.RegisterTendermintService(clientCtx)
 	}
 
+	if ctx.Viper.GetBool(srvflags.JSONRPCEnableMetrics) {
+		ethmetrics.Enabled = true
```

Comment:
I believe manually setting `Enabled` to true is not needed, just with the flag is enough as go-ethereum sets this variable in the init function of metrics.

Reply:
True